src/plugins/type/README.md

```diff
@@ -115,18 +115,23 @@ You can also change how values shall be restored in `kdbSet`:
 sudo kdb mount typetest.dump user/tests/type dump type booleans=#0 booleans/#0/true=true booleans/#0/false=false boolean/restoreas=#0
 ```
 
-The config key `boolean/restoreas` must be a valid index of the `booleans` array. If `boolean/restoreas` was set the chosen boolean pair
-will be used when values are restored in `kdbSet`. So in the above example the plugin accepts `1`, `true`, `0` and `false` as boolean values,
-on `kdbGet` it turns `true` into `1` and `false` into `0` and on `kdbSet` it turns `1` into `true` and `0` into `false`.
+The config key `boolean/restoreas` must be a valid index of the `booleans` array or the special value `none`. If
```

Comment:
What is the default?

Reply:
The default is described above the code snippet. By default we restore the value the user chose,  `boolean/restoreas` is only used to modify the behaviour.